packages/bluebird/src/abstracts/_variables-bootstrap.scss

```diff
@@ -259,17 +259,15 @@ $input-color-placeholder: $gray !default;
 
 //** Default `.form-control` height
 $input-height-base: (
-  $line-height-computed + ($padding-base-vertical * 2) + 2
+  $line-height-computed + ($padding-base-vertical * 2)
```

Comment:
Are you sure you want to remove the `+ 2`? I believe this is to account for the top/bottom border of form elements.